py/server/deephaven_internal/auto_completer/__init__.py

```diff
@@ -16,10 +16,10 @@
 later, we may add slow mode, which uses both static and interpreted completion modes.
 """
 
-from ._completer import Completer, Mode
+
+from ._completer import Completer, Mode, jedi_settings
 from jedi import preload_module, Interpreter
 
-jedi_settings = Completer()
```

Comment:
just curious... why move the instance location? the object is designed such that if customer wants to turn off autocomplete, they need to access this instance to set the mode to off.  Just want to make sure that is clear when you are making decisions about where things should live.